public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -131,15 +131,16 @@ class MetricsPanelCtrl extends PanelCtrl {
       }
 
       if (data.request) {
-        const { range, timeInfo } = data.request;
-        if (range) {
-          this.range = range;
-        }
+        const { timeInfo } = data.request;
         if (timeInfo) {
           this.timeInfo = timeInfo;
         }
       }
 
+      if (data.timeRange) {
```

Comment:
Found this place, but there could be several more. Would be better if one could be more certain where these updates are done.

Reply:
What about this one https://github.com/grafana/grafana/blob/6a699af27b8f6c8740cf9d640549c447acc95b9b/public/app/features/dashboard/panel_editor/QueryEditorRow.tsx#L98